README.md

```diff
@@ -167,5 +167,5 @@ The plan:
 
 ## Feature flags
 
-`localStorage.setItem('enableNewSwap', '1')` - enable refactored swap interface
```

Comment:
New swap is now enabled by default, removed this from readme

Reply:
Great you added it in the README